frontend/app/src/theme/community/windoge98.ts

```diff
@@ -31,6 +31,10 @@ export function getTheme(base: Theme): Theme {
     base.accent = magenta;
     base.disabledTxt = txt40;
     base.placeholder = txt60;
+    // base.font =

```

Comment:
Is this purposely left in?

Reply:
Yes - it's there as a suggestion but I don't want to impose too much on someone else's theme.